lib/pangram_cases.rb

```diff
@@ -1,15 +1,11 @@
 require 'exercise_cases'
 
-class PangramCase < OpenStruct
-  def name
-    'test_%s' % description.downcase.tr_s(" -'", '_').sub(/_$/, '')
-  end
-
+class PangramCase < ExerciseCase
   def workload
     [
     "phrase = '#{input}'",
     "    result = Pangram.pangram?(phrase)",
-    "    #{assertion} result, \"#{message}\""
+    "    #{assert} result, \"#{message}\""
```

Comment:
Doesn't work here (at least straight substitution doesn't). The interpolation in `message` would also need adjusting.